### PR TITLE
test: Add more data mapping tests

### PR DESCRIPTION
**Pull request by mutdmour**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: cypress/e2e/14-mapping.cy.ts [hunk 1]
```diff
@@ -34,4 +36,265 @@ describe('Data mapping', () => {
...
+				]
+			},
+		]);
+		ndv.actions.close();
+
+		canvasNode.actions.openNode('Set');
+		ndv.actions.switchInputMode('Table');
+		ndv.getters.inputDataContainer().get('table', { timeout: 10000 }).should('exist');
+
+		ndv.getters.nodeParameters().find('input[placeholder*="Add Value"]').click();
```
Comment (MiloradFilipovic) [inline cypress/e2e/14-mapping.cy.ts:71]: Nit: Can we add test id's or use getters in order to avoid such selectors in tests?

**Comment by MiloradFilipovic** [comment]
The third test (`maps expressions from json view`) is failing for me locally:
image

File: cypress/e2e/14-mapping.cy.ts [hunk 2]
```diff
@@ -34,4 +36,265 @@ describe('Data mapping', () => {
 		ndv.actions.mapDataFromHeader(2, 'value');
 		ndv.getters.inlineExpressionEditorInput().should('have.text', '{{ $json.timestamp }} {{ $json["Readable date"] }}');
 	});
+
+	it('maps expressions from table json, and resolves value based on hover', () => {
+		cy.fixture('Test_workflow-actions_paste-data.json').then((data) => {
+			cy.get('body').paste(JSON.stringify(data));
+		});
+		canvasNode.actions.openNode('Schedule Trigger');
+		ndv.actions.setPinnedData([
```
[the remark this answers is not in the document]
Comment (mutdmour) [reply cypress/e2e/14-mapping.cy.ts:45]: ya but I wanted to avoid this here..

File: cypress/e2e/14-mapping.cy.ts (hunk 1 above)
Comment (mutdmour) [reply cypress/e2e/14-mapping.cy.ts:71]: let me try

**Review by MiloradFilipovic — DISMISSED** [review]
Approving this since @alexgrozav confirmed that tests are passing for him locally

**Comment by mutdmour** [comment]
added skip e2e because running  tests separetly [here](https://github.com/n8n-io/n8n/actions/runs/4135122128/jobs/7147147642)

**Review by MiloradFilipovic — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.215.0`

**Pull request merged**